docs/source/actions.rst

```diff
@@ -52,10 +36,54 @@ Again, this should be a logical name.
 
 Note that all code called within this block is within the context of this action.
 While running the block of code within the ``with`` statement new actions created with ``start_action`` will get the top-level ``start_action`` as their parent.
-If there is no parent the action will be considered a task.
+
+
+Tasks: Top-level Actions
+------------------------
+
+A top-level action with no parent is called a task, the root cause of all its child actions.
+E.g. a web server receiving a new HTTP request would create a task for that new request.
+Log messages emitted from Eliot are therefore logically structured as a forest: trees of actions with tasks at the root.
 If you want to ignore the context and create a top-level task you can use the ``eliot.start_task`` API.
 
 
+.. _task fields:
+
+From Actions to Messages
+------------------------
+
+While the logical structure of log messages is a forest of actions, the actual output is effectively a list of dictionaries (e.g. a series of JSON messages written to a file).
+To bridge the gap between the two structures each output message contains special fields expressing the logical relationship between it and other messages:
+
+* ``task_uuid``: The unique identifier of the task (top-level action) the message is part of.
+* ``task_level``: The specific location of this message within the task's tree of actions.
+  For example, ``[3, 2, 4]`` indicates the message is the 4th child of the 2nd child of the 3rd child of the task.
```

Comment:
I guess this means that the initial message, `task_level=[1] action_type="parent" action_status="started"` is the first "child" of the task?